File: cpp-projects-2022/extra/translation/URI1607/URI1607.cpp
//
// Created by User on 17.09.2023.
//
#include "iostream"
#include "string"

using namespace std;

int main() {
    int n;
    cin >> n;
    for (int i = 0; i < n; ++i) {
        string firstWord;
        cin >> firstWord;
        string secondWord;
        cin >> secondWord;
        int transformation = 0;
        for (int j = 0; j < firstWord.length(); ++j) {
            int counter = tolower(firstWord[j]) - 'a';
            int counter2 = tolower(secondWord[j]) - 'a';
            int gap = counter2 - counter;
            if (gap < 0) {
                gap += 26;
            }
            transformation += gap;
        }
        cout << transformation << endl;
    }
    return 0;
}

File: java-projects-2022/uri-midterm/src/URI1607.java
import java.util.Scanner;

public class URI1607 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        for (int i = 0; i < n; ++i) {
            String firstWord = input.next();
            String secondWord = input.next();
            int transformation = 0;
            for (int j = 0; j < firstWord.length(); ++j) {
                int counter = Character.toLowerCase(firstWord.charAt(j)) - 'a';
                int counter2 = Character.toLowerCase(secondWord.charAt(j)) - 'a';
                int gap = counter2 - counter;
                if (gap < 0) {
                    gap += 26;
                }
                transformation += gap;
            }
            System.out.println(transformation);
        }
    }
}
